api/controller/__init__.py

```diff
@@ -1,4 +1,8 @@
-from pdb import set_trace
+from nose.tools import (
+    assert_raises,
+    eq_,
+    set_trace,
+)
```

Comment:
I know set_trace is useful here but why import eq_ and assert_raises, which are only used in tests?